amd_openvx/openvx/hipvx/CMakeLists.txt

```diff
@@ -35,17 +35,12 @@ list(APPEND HIPVX_SOURCES
 
 set_source_files_properties(${HIPVX_SOURCES} PROPERTIES HIP_SOURCE_PROPERTY_FORMAT 1)
 
-set(COMPILER_FOR_HIP ${ROCM_PATH}/bin/hipcc)
+set(HIP_CXX_FLAGS -std=gnu++14)
+set(COMPILER_FOR_HIP ${ROCM_PATH}/llvm/bin/clang++)
 set(CMAKE_CXX_COMPILER ${COMPILER_FOR_HIP})
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${HIP_CXX_FLAGS}")
 
-add_library(openvx_hip OBJECT ${HIPVX_SOURCES})
-set_target_properties(openvx_hip PROPERTIES POSITION_INDEPENDENT_CODE ON)
-target_compile_definitions(openvx_hip PRIVATE __HIP_PLATFORM_AMD__)
-
-if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
```

Comment:
@asalmanp are we leaving out future HIP support on Windows?

Reply:
@kiritigowda HIP on Windows doesn't use the MSVC. It still uses the HIP compiler for generating the HIP device code. This CMake is specific for generating the OpenVX HIP Kernels only.